src/dbcp/validation/tests.py

```diff
@@ -243,9 +243,10 @@ def test_county_wide_coverage(engine: Engine):
         df.shape[0] == n_counties
     ), "counties_wide_format does not contain all counties"
     notnull = df.notnull()
-    assert (
-        notnull.any(axis=1).sum() == 2457
-    ), f"counties_wide_format has unexpected county coverage: {notnull[notnull.any(axis=1)]}"
+    assert notnull.any(axis=1).sum() == 2461, (
```

Comment:
Any idea why this went up?

Reply:
The PUDL update included about 300 more rows in `core_eia__entity_plants` for some reason. Apparently some of those were in new counties, or the county assignments changed. I didn't make any code changes to that pipeline